Entities/PieMenu.cpp

```diff
@@ -630,12 +637,12 @@ namespace RTE {
 			SetEnabled(false);
 		}
 
-		return false;
+		return true;
```

Comment:
Why do you return true here, when there's no input??
That'll constantly reset the hover timer, making things not deselect after a delay.

Reply:
Whoops. That was a leftover from when I was trying to do things a little differently. Good catch